client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/jetpack-search.tsx

```diff
@@ -0,0 +1,73 @@
+import { Button } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import ThankYouV2 from 'calypso/components/thank-you-v2';
+import ThankYouProduct from 'calypso/components/thank-you-v2/product';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { useSelector } from 'calypso/state';
+import {
+	getJetpackSearchCustomizeUrl,
+	getJetpackSearchDashboardUrl,
+} from 'calypso/state/sites/selectors';
+import { getSelectedSiteId, getSelectedSite } from 'calypso/state/ui/selectors';
+
+export default function JetpackSearchThankYou() {
```

Comment:
`<JetpackSearchThankYou>` uses the redesigned component `<ThankYouV2>` to render the Congrats page for Jetpack Search. I basically took the copy and logic from `checkout/checkout-thank-you/header.jsx`, plugged them into `<ThankYouV2>`.